src/lib/common/include/sol-mainloop-contiki.h

```diff
@@ -39,7 +39,26 @@
 extern "C" {
 #endif
 
+/**
+ * Set the event and data that woke up the Soletta protothread,
```

Comment:
@brief is needed here. Leave a blank line after handled. so the "should" isn't part of @brief.